_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: otel_logs_source 
+parent: Sources
+grand_parent: Pipelines
+nav_order: 25
+---
+
+# otel_logs_source
+
+
+The `otel_logs_source` source is an OpenTelemetry source that follows the [OpenTelemetry Protocol Specification](https://github.com/open-telemetry/oteps/blob/master/text/0035-opentelemetry-protocol.md), receives logs from the OTel Collector in the form of `ExportLogsServiceRequest` records.
+
+This source supports the `OTLP/gRPC` protocol.
+{: .note}
+
+## Configuration
+
+You can configure the `otel_logs_source` source with the following options. 
+
+| Option | Type | Description |
+| :--- | :--- | :--- |
+| port | int | Represents the port that the `otel_logs_source` is running on. Default value is `21892`. |
+| path | string | Represents the path for sending unframed HTTP requests. You can use this option to support unframed gRPC with HTTP idiomatic path to a configurable path. The path should start with `/` and length should be at least 1. `/opentelemetry.proto.collector.logs.v1.LogsService/Export` endpoint is disabled for both gRPC and HTTP requests if the path is configured. The path can contain `${pipelineName}` placeholder, which is replaced with pipeline name.  If the value is empty and `unframed_requests` is `true`, then the path that the source provides is `/opentelemetry.proto.collector.logs.v1.LogsService/Export`. | 
```

Comment:
Confirm that my changes retain the intended meaning.

Reply:
Confirmed, the intended meaning is in tact. Thanks!